src/test/java/uk/gov/companieshouse/web/pps/util/PenaltyUtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.web.pps.util;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PenaltyUtilsTest {
+
+    @Autowired
```

Comment:
Don't expect us to need the Autowired annotation here if it is just a unit test.
Think it should be removed.